project/V.scala

```diff
@@ -92,9 +92,9 @@ object V {
 
   // Scala 3
   def nonDeprecatedScala3Versions =
-    Seq(nextScala3RC, scala3, "3.1.1", "3.1.0", "3.0.2")
+    Seq(nextScala3RC, scala3, "3.1.1", "3.1.0")
   def deprecatedScala3Versions =
-    Seq("3.1.3-RC4", "3.1.3-RC3", "3.1.3-RC2", "3.0.1", "3.0.0")
+    Seq("3.1.3-RC4", "3.1.3-RC3", "3.1.3-RC2", "3.0.2", "3.0.1", "3.0.0")
```

Comment:
hm, I'm not sure that I fully got what you mean by `mark the current minor release as "non deprecated"`.

What about deprecation of `3.0.2`. Currently, we can say that ecosystem is switched on 3.1.x so this is a valid deprecation.
I think we can fully drop the support of 3.0.x after 3.2.0 will be released

Reply:
Right now it's still not clear how it will be exactly. There is a [wip-document](https://github.com/scala/scala-lang/pull/1387/files) about introducing Scala LTS and Scala Next. However, I wouldn't say that I have a clear picture what to do with munit at the moment when there will be more than one LTS versions.

What about `3.2` [there is a conversation](https://github.com/scala/scala-lang/pull/1387/files#r888378274) and as I got the transition on it will be the same as with 3.0->3.1.

Reply:
>  So in theory in the future when 3.3 lands and that's deemed LTS, we'll no longer support 3.0, 3.1 and 3.2 will probably be deprecated and then we'll focus on LTS and Next support?

Probably. However, I had a felling than I want to return `major.minor` prefixes back.